crates/noirc_frontend/src/hir/def_map/aztec_library.rs

```diff
@@ -0,0 +1,424 @@
+use acvm::FieldElement;
+use noirc_errors::Span;
+
+use crate::{
+    token::Attribute, BlockExpression, CallExpression, CastExpression, Distinctness, Expression,
+    ExpressionKind, ForExpression, FunctionReturnType, Ident, IndexExpression, LetStatement,
+    Literal, MethodCallExpression, NoirFunction, ParsedModule, Path, Pattern, Statement,
+    UnresolvedType, UnresolvedTypeData, Visibility,
+};
+
+//
+//             Helper macros for creating noir ast nodes
+//
+macro_rules! ident {
+    ($name:expr) => {
+        Ident::new($name.to_string(), Span::default())
+    };
+}
+
+macro_rules! ident_path {
+    ($name:expr) => {
+        Path::from_ident(ident!($name))
+    };
+}
+
+macro_rules! expression {
+    ($kind:expr) => {
+        Expression::new($kind, Span::default())
+    };
+}
```

Comment:
This code is temporary so I won't block on it, but in the future these can just be functions